.github/workflows/build-preview-environment.yml

```
@@ -0,0 +1,108 @@
+      registry: ghcr.io/${{ github.repository_owner }}
+      context: apps/workflows-dashboard
+      image_name: workflows-dashboard
+      ref: ${{ github.event.pull_request.head.sha }}
+      tag: ${{ needs.deploy-dev-pr-environment.outputs.env_name }}
+
+  deploy-preview:
+    needs: [deploy-dev-pr-environment,build-wf-service,build-backoffice,build-kyb,build-dashboard]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger workflow in another repo
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)